docs/source/documentation/platform/infrastructure/run-data-product-in-platform.html.md.erb

```diff
@@ -1,48 +1,65 @@
 ---
 owner_slack: "#data-platform-notifications"
-title: Run data product in Data Platform (POC)
-last_reviewed_on: 2023-04-04
+title: Data as a Product in Data Platform (PoC)
+last_reviewed_on: 2023-09-27
 review_in: 6 months
 weight: 15
 ---
 
 # <%= current_page.data.title %>
 
-## Context
-We want a compute process that is scalable and event driven to run the data product. The data product will be sent to Data Platform for processing. After processing, the data should be available in a suitable format accessible to data consumers. The processing will include receiving raw data, applying transformations, cleansing and finally storing data in a correct location and format.
+## Data as a Product: Vision
 
-<strong>The architecture will be built in the `data-platform-development`, a sandbox account within MoJ's Modernisation Platform, which is deployed using the [modernisation-platform-environments](https://github.com/ministryofjustice/modernisation-platform-environments) repository</strong>
+Each Data Product will require a [specification](https://github.com/ministryofjustice/data-platform-products/blob/main/data-products/_docs/product-specification.md) (still under refinement) alongside [other defined pieces of descriptive metadata captured at registration](https://github.com/ministryofjustice/data-platform-products/tree/main/data-products#defining-a-data-product).
 
-## Data Producer Configuration Flow
+Data Products:
 
-![Data platform general diagram](../images/data-platform-configuration-flow.png "Data platform configuration Flow")
+* May be multi-component, e.g. consisting of multiple tables
+* Can consist of structured or unstructured data (or a combination)
+* Are controlled and updated by Data Owners, Data Stewards, and Data Custodians.
```

Comment:
Suggest just "**Are controlled and updated by Data Owners**" for now, as you're right, I'm fairly sure we haven't defined the Data Steward and Data Custodian roles. In fact those definition's are really up to the DIP team, so let's not create a blocker waiting for those.